libs/wire-api/src/Data/Swagger/Typed.hs

```diff
@@ -0,0 +1,468 @@
+{-# LANGUAGE DeriveFunctor #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Data.Swagger.Typed
```

Comment:
Perhaps you could add a module comment here which links to the tutorial docs?

Reply:
Done.